css/airbnb.css

```
@@ -617,6 +616,321 @@ h2{color:black;
 
 a {text-decoration: none;}
```

여기서 css 컨벤션을 통일 시켜봅시다
```suggestion
a {
    text-decoration: none;
}
```